editor/editor_node.cpp

```diff
@@ -2099,6 +2099,29 @@ void EditorNode::edit_item(Object *p_object) {
 	}
 }
 
+void EditorNode::store_object_properties(Object *p_object) {
```

Comment:
I think you cold put this in InspectorDock rather than EditorNode to avoid further bloating this.

Reply:
I think the function could also be renamed to store_script_properties?